peer/brontide.go

```diff
@@ -479,8 +485,7 @@ func (p *Brontide) Start() error {
 		return nil
 	}
 
-	peerLog.Tracef("Peer %v starting with conn[%v->%v]", p,
-		p.cfg.Conn.LocalAddr(), p.cfg.Conn.RemoteAddr())
+	peerLog.Tracef("Peer %v starting", p)
```

Comment:
`String` doesn't have `p.cfg.Conn.LocalAddr()` in it fwiw, so we're losing a little information here.

Reply:
Inadvertent change during a rebase, reverted.